.travis.yml

```diff
@@ -56,8 +56,8 @@ jobs:
       script:
         # check python code
         ## check syntax errors and fail the build if any are found.
-        - flake8 .
-
+        # TODO: remove extend-exclude when we implement resources
+        - flake8 . --extend-exclude="monkey/monkey_island/cc/repository, monkey/monkey_island/cc/models/networkmap.py"
```

Comment:
Flake8 complains about unused classes or what?

Reply:
Yep. We don't have `Arc` or `Machine` classes.

Reply:
Add `# noqa:` comments into the code instead of doing this. We'll have to remember to undo this change, whereas when we modify the code to implement things, we're more likely to remember to remove the comments.